### PR TITLE
Add series details to feed

### LINKED ISSUE #248: Add series details to OPDS feed

### PR DESCRIPTION
**Pull request by courte**
This branch adds the series name and a work's position in the volumes of a series to the OPDS feed. It also removes some bits of code that weren't actually being used.

Fixes #248.

### CODE REVIEW

File: opds.py [hunk 1]
```diff
@@ -222,6 +218,18 @@ def authors(cls, work, license_pool, edition, identifier):
...
 
     @classmethod
+    def series(cls, series_name, series_position):
+        """Generate a schema:Series tag for the given name and position."""
+        if not series_name:
+            return None
+        series_details = dict()
+        series_details['name'] = series_name
+        if series_position:
+            series_details['{%s}position' % schema_ns] = unicode(series_position)
```
Comment (aslagle) [inline opds.py]: what's the reason for converting to unicode? it seems like it would be more useful as an integer.

Comment (courte) [inline opds.py]: The element maker expects a string. Trying to pass an integer raises an error. 😞

Comment (aslagle) [inline opds.py]: oh well :(

**Comment by aslagle** [comment]
:+1:

**Pull request merged**